components/brave_wallet_ui/common/hardware/ledgerjs/fil_ledger-untrusted-transport.ts

```diff
@@ -0,0 +1,158 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Should the name of this file be 'fil-ledger-untrusted-transport.ts' so it matches with 'eth-ledger-untrusted-transport.ts' and 'sol-ledger-untrusted-transport.ts'?

To be honest, I'm not sure why we are using undescores for the '*_ledger_bridge_keyring' files.  I suspect that might have been a mistake originally that has now become convention.  I'd be fine with using dashes '-' instead of underscores '_' across the board.